test/run-tests.sh

```diff
@@ -26,19 +26,12 @@ done
 # to the day before, to reduce the amount of refs being transferred and thus speed up CI.
 
 test_downstream() { # owner/repo shallow-since commit make-target
-	if [ ! -d ${1##*/} ]; then
-		git clone https://github.com/$1.git --shallow-since=$2 --single-branch
-	fi
 	pushd ${1##*/}
```

Comment:
```suggestion
	if ! pushd ${1##*/}; then
		echo >&2 'Please run `fetch-test-deps.sh` before running the test suite'
		return 1
	fi
```
Hinting the user at how to fix the problem should be better UX. Doing this on `pushd` fail is preferable, as they might have e.g. symlink'd the directories (I think).

Reply:
Very good point, I haven't thought about giving an error message for users that will run the scripts manually. Thanks for the suggestion.